docs/framework/network-programming/managing-connections.md

```diff
@@ -22,6 +22,9 @@ ms.assetid: 9b3d3de7-189f-4f7d-81ae-9c29c441aaaa
 # Managing Connections
 
 Applications that use HTTP to connect to data resources can use the .NET Framework's <xref:System.Net.ServicePoint> and <xref:System.Net.ServicePointManager> classes to manage connections to the Internet and to help them achieve optimum scale and performance.  
+
+> [!NOTE]
+> `ServicePoint` and `ServicePointManager` are considered legacy and are only stubbed out in .NET Core, .NET 5, and later versions. Very little is used by any of the networking stack. They are referenced from `HttpWebRequest` (also legacy) in only a few places, in an attempt to respect some globally-set settings. `ServicePoint` doesn't track anything about current connections used by `HttpClient` (or `HttpWebRequest`, which layers on top of it) and is hardcoded to return 0 for <xref:System.Net.ServicePoint.CurrentConnections%2A>.
```

Comment:
```suggestion
> `ServicePoint` and `ServicePointManager` are considered legacy and and exist only as stubs in .NET Core, .NET 5, and later versions. Very little is used by any of the networking stack. They are referenced from `HttpWebRequest` (also legacy) in only a few places, in an attempt to respect some global settings. `ServicePoint` doesn't track anything about current connections used by `HttpClient` (or `HttpWebRequest`, which layers on top of it) and is hardcoded to return 0 for <xref:System.Net.ServicePoint.CurrentConnections%2A>.
```

The way it was written, it could be read as that those two types are only stubbed out in core/5+ (but not in earlier versions).